src/content/en/2020/css.md

```diff
@@ -94,18 +103,20 @@ Another metric of size is the number of rules. The median page carries a total o
 
 ## Selectors and the cascade
 
+{# TODO (Authors) We try to avoid empty headrs. Could we have a short intro sentence or two here? #}
```

Comment:
@LeaVerou, @svgeesus, @rachelandrew - could one of you add an intro to this section here?

Reply:
A went with this. Feel free to shout if you disagree:

> CSS offers a number of ways of apply styles to page, from classes, ids and using the all-important cascade to avoid duplicating styles. So how are developers applying their styling to their pages?